frontend/public/components/utils/storage-class-dropdown.tsx

```diff
@@ -8,7 +8,7 @@ import { isDefaultClass } from '../storage-class';
 
 /* Component StorageClassDropdown - creates a dropdown list of storage classes */
 
-class StorageClassDropdown_ extends React.Component<StorageClassDropdownProps, StorageClassDropdownState> {
+export class SCDropdown extends React.Component<StorageClassDropdownProps, StorageClassDropdownState> {
```

Comment:
Exporting both `StorageClassDropdown` and `SCDropdown` is confusing. You need to give better names to make the difference clear.